classes/Compatibility/backword-compatibility.php

```diff
@@ -16,6 +16,11 @@
  *  @copyright Since 2019 Shopping Feed
  *  @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
  */
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
 if (version_compare(phpversion(), '7', '<')) {
```

Comment:
The module is not compatible with PHP 7- now.
I think we can remove this file and the call of this file.